gemrb/core/Audio.h

```diff
@@ -111,7 +111,7 @@ class GEM_EXPORT Audio : public Plugin {
 	virtual bool Stop() = 0;
 	virtual bool Pause() = 0;
 	virtual bool Resume() = 0;
-	virtual int CreateStream(Holder<SoundMgr>) = 0;
+	virtual int CreateStream(std::shared_ptr<SoundMgr>&) = 0;
```

Comment:
let's not pass a shared_ptr by reference to another thread. instead `std::move` at the place of assignment.